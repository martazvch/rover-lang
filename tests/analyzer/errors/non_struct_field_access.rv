fn main() {
    var a = 1
    print a.go
}

