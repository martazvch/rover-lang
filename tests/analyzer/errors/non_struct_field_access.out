non_struct_field_access, int
